applications/FSIapplication/custom_utilities/AdvancedNMPointsMapper.cpp

```diff
@@ -889,7 +889,7 @@ void AdvancedNMPointsMapper::NormalVectorToScalarMap(const Variable<array_1d<dou
 
         MInterp = prod(MCons, inv_aux_GPPos); // Interpolation matrix (NodalValues = (L/6)*MInterp*GaussValues)
 
-        std::vector< boost::shared_ptr<array_1d<double,6> > > pInterpValues;
```

Comment:
to be honest i am a little surprised that all of these vectors of shared_ptr s are needed... in any case the change is for good so i'll accept it

Reply:
Taking into account that this mapper is to be substituted by the ones in the MappingApplication I wouldn't take care of this kind of things.